.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@1.8.0
+
+jobs:
+  build:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run: gradle build
+  unitTests:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run: gradle test
+  acceptanceTests:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run:
+          command: gradle run
+          background: true
+      - run: git submodule update --init --recursive
+      - ruby/install:
+          version: '2.6.8'
+      - run:
+          command: |
+            cd http_server_spec
+            gem install bundler
+            bundle install --path vendor/bundle
+            sudo apt install lsof
+            rake test:f1
+
+workflows:
+  build_and_test:
+    jobs:
+      - build
+      - unitTests:
+          requires:
+            - build
+      - acceptanceTests:
+          requires:
+            - build
```

Comment:
Not related to the change. Can you try running the pipeline with a failing acceptance test?

Reply:
This is a really good idea, I will for sure try running it with a failing test and post the results here!

Reply:
Acceptance tests fail stopping circleci [in this commit](https://github.com/jonmanahan/waiter/pull/16/commits/881f234bc04b17366de37d3e832bce3f1d6b3fd0), unit tests fail stopping circle ci [in this commit](https://github.com/jonmanahan/waiter/pull/16/commits/c9494704faa34c9a0522cdecfccba11334979681)

Reply:
Looks like the build step also runs unit test and will fail if any unit tests fails. Maybe we should just have the build step be `build and unit test` then we don't need a separate step for unit tests.

Reply:
will do!